src/main/docs/guide/kubernetes-client-openapi.adoc

```diff
@@ -0,0 +1,52 @@
+The `micronaut-kubernetes-client-openapi` module uses Micronaut Netty HTTP Client and generated apis and modules from https://github.com/kubernetes-client/java/blob/master/kubernetes/api/openapi.yaml[Kubernetes Client OpenAPI Spec].
```

Comment:
at a description why you would want to use this client instead of the Kubernetes official one. Reasons that come to mind include:

* No extra dependencies needed (okhttp, Kotlin etc.)
* Unified configuration with Micronaut HTTP client
* Support for plugging in filters etc.
* Native Image compatibility

Reply:
Done